### DISCUSSION TITLE
Android gdscript writing keyboard hides writing.

### DISCUSSION THREAD

**Discussion by MarkoGrbec** [godotengine/godot-proposals#10715] [Platforms]
Well when I try to type something my keyboard is so large that it hides 1/2 screen I don't see what I'm typing.

When keyboard is active we should be able to drag the screen to see what is on the other side (below keyboard)

Right slider is not available at beginning cuz of too little lines.

Also at making nodes it hides the text.

Maybe but it would be hard to implement is shrink full editor.
Another idea is to make godot keyboard but it's time consuming.

**Comment by Zireael07** [comment]
This is a bug, not discussion material

Reported at https://github.com/godotengine/godot/issues/76803

**Comment by AThousandShips** [comment]
Consolidating in the bug report above
